src/main/java/org/apache/brooklyn/cloudfoundry/entity/VanillaCloudfoundryApplicationImpl.java

```diff
@@ -101,6 +95,16 @@ private void initApplicationName() {
     }
 
     @Override
+    public Class getDriverInterface() {
+        return VanillaPaasApplicationDriver.class;
+    }
+
+    @Override
+    public VanillaPaasApplicationDriver getDriver() {
+        return driver;
+    }
+
+    @Override
     protected void initEnrichers() {
```

Comment:
I found similar called in `SoftwareProcessImpl` ([here](https://github.com/apache/brooklyn-server/blob/0b9682b8830df7715bb8b579cfc40a1734f03038/software/base/src/main/java/org/apache/brooklyn/entity/software/base/SoftwareProcessImpl.java#L142)) and I decided to follow its example